grid-proxy/cmds/proxy_server/main.go

```diff
@@ -96,12 +101,24 @@ func main() {
 	}
 	defer sub.Close()
 
-	relayClient, err := createRMBClient(ctx, f.relayURL, f.mnemonics, sub)
+	relayRPCClient, err := createRPCRMBClient(ctx, f.relayURL, f.mnemonics, sub)
 	if err != nil {
 		log.Fatal().Err(err).Msg("failed to create relay client")
 	}
 
-	s, err := createServer(f, GitCommit, relayClient)
+	db, err := db.NewPostgresDatabase(f.postgresHost, f.postgresPort, f.postgresUser, f.postgresPassword, f.postgresDB)
+	if err != nil {
+		log.Fatal().Err(err).Msg("couldn't get postgres client")
+	}
+
+	worker, err := gpuindexer.NewNodeGPUIndexer(ctx, f.relayURL, f.mnemonics, sub, db, f.indexerCheckIntervalMins, f.workerBatchSize)
```

Comment:
This is a little bit misleading, It should return the concrete Indexer and on the indexer you do indexer.StartWorkers .. etc

Reply:
The name of the variable is misleading, this is the indexer, will change the name